rest-service/manager_rest/workflow_executor.py

```diff
@@ -113,6 +118,17 @@ def execute_system_workflow(wf_id,
                          context=context, execution_creator=execution_creator)
 
 
+def generate_execution_token(execution_id):
+    sm = get_storage_manager()
+    execution = sm.get(models.Execution, execution_id)
+    execution_token = uuid.uuid4().hex
+
+    # Store the token hashed in the DB
+    execution.token = hash_data(execution_token)
```

Comment:
tbh there's not much point hashing a uuid4 i don't think, so if we wanted to, we could skip that...